### DISCUSSION TITLE
search for explorer

### DISCUSSION THREAD

**Discussion by Adnan0061** [hasura/graphql-engine#8660] [General]
It would be nice if there were a search box for the explorer options.
for big projects, it is a mess if you have to select anything instead of writing manually.
![Screenshot 2022-07-02 190510](https://user-images.githubusercontent.com/48943431/177002167-741ee704-3673-448f-a07b-81ed8237b8a6.jpg)

**Comment by meetzaveri** [comment]
Hello,

Thanks for presenting your idea. We would recommend you to **create a github issue** on this with subject of feature request and follow it. That would be a good starting point and our team would be aware of it for tracking it in the future. You can easily do this via clicking [here](https://github.com/hasura/graphql-engine/issues/new?created_from_discussion_number=8660) or clicking "Create issue from discussion" which is present in the bottom of the right sidebar of this discussion page.

Also, we don't know if you would be aware of, but there's a way if you want to find schema names of queries, mutations or subscriptions. If you see in that GraphiQL editor, in extreme left top corner there is "Docs" button.

![captures_chrome-capture-2022-6-3 (1)](https://user-images.githubusercontent.com/21293048/177023481-fdc22724-d0f6-49d5-af59-f6e90a3a44a0.png)

 If you click that, documentation explorer would open. Where you can search query/mutation root names.
 
![captures_chrome-capture-2022-6-3](https://user-images.githubusercontent.com/21293048/177023424-d4db90f1-d59a-4756-99a4-c58841d4124a.png)

**Reply by Adnan0061** [reply]
about the docs you mentioned, 
we still need to find the root from explorer and then select the option, which is more repetitive if I know the root name but need to choose options from that root, isn't it?

it would be so much better if I could search and only the filtered results will show on the explorer.

**Reply by meetzaveri** [reply]
Understood. Well, then I would recommend going ahead and raise an issue for feature request.

**Reply by Adnan0061** [reply]
@meetzaveri I have created this issue from your link but it actually not went to the feature request section. can you add the label or transfer it to the feature request issue?

**Reply by meetzaveri** [reply]
Sure, I've added necessary label(s) for it.
